eos-updater/poll.c

```diff
@@ -370,6 +379,33 @@ check_for_update_using_booted_branch (OstreeRepo           *repo,
   return TRUE;
 }
 
+static gboolean
+get_booted_refspec_from_default_booted_sysroot_deployment (gchar               **out_refspec,
+                                                           gchar               **out_remote,
+                                                           gchar               **out_ref,
+                                                           OstreeCollectionRef **out_collection_ref,
+                                                           GError              **error)
+{
```

Comment:
I know :(

I was thinking the same thing with caching but I wanted to avoid being too invasive. I think it'd be better to do this in a follow up - and preferably by someone who is more familiar with the trickier stuff like eol-rebase and P2P.

Reply:
Yes, this should be done in a follow-up.